src/status_im/ui/screens/chat/styles/input/send_button.cljs

```diff
@@ -3,14 +3,15 @@
 
 (defn send-message-container [rotation]
   {:background-color colors/blue
-   :width            30
-   :height           30
-   :border-radius    15
-   :margin           10
-   :padding          4
-   :margin-left      8
-   :margin-bottom    11
-   :transform        [{:rotate rotation}]})
+   :width              30
+   :height             30
+   :padding-vertical   3
+   :padding-horizontal 4
+   :border-radius      15
+   :margin             10
+   :margin-left        8
+   :margin-bottom      11
+   :transform          [{:rotate rotation}]})
 
 (def send-message-icon
   {:height 22
```

Comment:
hey @yenda shouldn't all our icons be 24px in size, does 22 here mean we scale down this one?

Reply:
this style is not used anymore I forgot to remove it

Reply:
I've noticed it because the 30 width, height are correct, padding should be a 3 on all sides, here it's 4 horizontal.